lib/simple_google_auth.rb

```diff
@@ -33,7 +33,9 @@ def self.uri(state)
       response_type: "code",
       client_id:     config.get_or_call(:client_id),
       redirect_uri:  config.redirect_uri,
-      state:         state
+      access_type:   "offline",
+      state:         state,
+      approval_prompt: "force"
```

Comment:
(or maybe have some configuration option for offline – I don't know what approval_prompt does exactly, but it seems like it should be at the gem user's discretion)